### ISSUE TITLE
Panel's views titles missing in SDK static mode 

### ISSUE THREAD

**Issue by trepmag** [unep-grid/map-x-mgl#803]
Hello, this was bring to my attention by @antobenve. Indeed, see this in live for a specific view (MX-T0DOH-N3IVM-FFXSS):
- normal static display: https://app.mapx.org/static.html?views=MX-T0DOH-N3IVM-FFXSS&language=en
- sdk static mode: https://codepen.io/trepmag/pen/YzErKjw

Either I missed an options or there is a discrepancy behavior btw both?...

Screenshot:
![image](https://user-images.githubusercontent.com/875021/153899743-72268460-16eb-4677-a22c-f4892015130b.png)

**Comment by fxi** [comment]
Yep, the title is missing when dynamically adding views in static mode, using the SDK.

**Comment by thomaspiller** [comment]
Tested on staging and the titles are now present. Thanks.
